spec/features/create_destroy_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe "Create and Destroy Comment" do
+
+  describe "successful" do
+    it "Signs the user up, in, makes a list, then adds and deletes an Item through BDD" do
```

Comment:
If a spec is flagged with `js: true`, Capybara knows that the browser it uses for testing should be capable of handling javascript.

Is the functionality under test here using AJAX now? If yes, try adding `js: true` spec metadata to the end of line 6 so that Capybara uses a JavaScript-capable browser in the tests. Capybara's default browser ignores JavaScript.

The metadata looks like this:

```ruby
  it "something something something", js: true do
    ...
  end
```
The `js: true` metadata can be added to `describe`, `scenario`, and `context` blocks too. 

Do this on a separate branch as there may be some additional setup needed to get Capybara's JavaScript browser working before this is ready to be used.